README.md

```diff
@@ -31,16 +32,29 @@
+Semgrep is a fast, open-source, static analysis tool for finding bugs and enforcing code standards at editor, commit, and CI time. [Get started →.](#getting-started)
+
+Semgrep runs fully on your computer or build environment: **your code is never sent anywhere**.
+
+Its rules look like the code you already write; no abstract syntax trees, regex wrestling, or painful DSLs. Here's a quick rule for finding Python `print()` statements, run it online in Semgrep's Playground by clicking the image:
 
-The [Semgrep Registry](https://semgrep.dev/explore) has 1,000+ rules written by the Semgrep community covering security, correctness, and performance bugs. No need to DIY unless you want to.
+<p align="center">
+    <a href="https://semgrep.dev/s/ievans:print-to-logger"><img src="https://github.com/returntocorp/semgrep/blob/dlukeomalley/readme-intro/doc/playground-example.png" height="281px" alt="Semgrep rule example for finding Python print() statements"/></a>
+</p>
 
-Semgrep is used in production everywhere from one-person startups to multi-billion dollar companies; it’s the engine inside tools like [NodeJsScan](https://semgrep.dev/p/nodejsscan). See tools [powered by Semgrep](https://semgrep.dev/docs/integrations/#semgrep-as-an-engine).
+The Semgrep ecosystem includes:
 
-Semgrep is developed and commercially supported by [r2c, a software security company](https://r2c.dev). r2c’s hosted service, [Semgrep App](https://semgrep.dev), lets organizations easily deploy in CI, manage rules across many projects, monitor the efficacy of code policy, and integrate with 3rd-party services. r2c offers free and paid hosted tiers ([see pricing](https://r2c.dev/pricing)).
+- Semgrep - this project, the open-source command line tool at the heart of everything
+- [Semgrep CI](https://semgrep.dev/docs/semgrep-ci/) - a specialized Docker image for running Semgrep in CI environments
+- [Semgrep Playground](https://semgrep.dev/editor) - an online interactive editor for writing and sharing rules
+- [Semgrep Registry](https://semgrep.dev/explore) - 1,000+ community-driven rules covering security, correctness, and performance bugs
+- [Semgrep App](https://semgrep.dev/manage) - deploy, manage, and monitor Semgrep at scale with free and paid tiers.
 
-### Language support
+Semgrep is used by 100,000+ developers a month and powers tools like [GitLab SAST](https://docs.gitlab.com/ee/user/application_security/sast/#supported-languages-and-frameworks), [NodeJsScan](https://semgrep.dev/p/nodejsscan), and [Salus](https://github.com/coinbase/salus/blob/master/docs/scanners/semgrep.md). See [tools powered by Semgrep](https://semgrep.dev/docs/extensions//#semgrep-as-an-engine).
```

Comment:
I feel like listing companies would actually be better here

Reply:
+1 to listing companies

Reply:
Updated. Let me know what you think.